projects/igniteui-angular/src/lib/expansion-panel/expansion-panel.component.ts

```diff
@@ -192,7 +199,7 @@ export class IgxExpansionPanelComponent extends ToggleAnimationPlayer implements
 
     private _collapsed = true;
 
-    constructor(private cdr: ChangeDetectorRef, protected builder: AnimationBuilder) {
+    constructor(private cdr: ChangeDetectorRef, private elementRef: ElementRef, protected builder: AnimationBuilder) {
```

Comment:
Adding a new parameters to a public class's constructor is a breaking change.
Though I highly doubt that would affect anyone, `elementRef` could still be moved as the last parameter and be made options (`?`) so as to avoid it.